platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerViewModel.swift

```diff
@@ -203,6 +209,54 @@ public extension WysiwygComposerViewModel {
     func treeRepresentation() -> String {
         model.toTree()
     }
+
+    /// Set a mention with `At` pattern. Usually used
+    /// for mentioning a user.
+    ///
+    /// - Parameters:
+    ///   - link: The link to the user.
+    ///   - name: The display name of the user.
+    func setAtMention(link: String, name: String) {
+        let update: ComposerUpdate
+        if let suggestionPattern, suggestionPattern.key == .at {
+            update = model.setLinkSuggestion(link: link, text: name, suggestion: suggestionPattern)
+        } else {
+            _ = model.setLinkWithText(link: link, text: name)
+            // FIXME: remove this if Rust adds this space for free
```

Comment:
I thought that the rust model add this at the moment? Or is this replacing an nbsp with a `" "` or something like that?

Reply:
The Rust model adds it on the `set_link_suggestion` but not on `set_link_with_text`.
Will make this change in a subsequent PR to avoid more noise on this one.

Reply:
(on the latter, it just adds nothing)